manifests/cluster-network-addons/0.12.0/network-addons-config-example.cr.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  imagePullPolicy: IfNotPresent
+  kubeMacPool: {}
```

Comment:
"kubeMacPool": {
              "rangeStart": "02:00:00:00:00:00",
              "rangeEnd": "FD:FF:FF:FF:FF:FF"
            },

Reply:
It is omitted on purpose. By default user doesn't care about the range, they care that there are no collisions. If they want to have a specific range, they can follow README

Reply:
Fair enough